tests/store/model_registry/test_sqlalchemy_store.py

```diff
@@ -972,6 +969,47 @@ def search_versions(filter_string):
         assert mvds[0].source == "A/B"
         assert mvds[0].description == "Online prediction model!"
 
+    def test_search_model_versions_by_tag(self):
+        # create some model versions
+        name = "test_for_search_MV_by_tag"
+        self._rm_maker(name)
+        run_id_1 = uuid.uuid4().hex
+        run_id_2 = uuid.uuid4().hex
+
+        mv1 = self._mv_maker(
+            name=name,
+            source="A/B",
+            run_id=run_id_1,
+            tags=[ModelVersionTag("t1", "abc"), ModelVersionTag("t2", "xyz")],
+        )
+        assert mv1.version == 1
+        mv2 = self._mv_maker(
+            name=name,
+            source="A/C",
+            run_id=run_id_2,
+            tags=[ModelVersionTag("t1", "abc"), ModelVersionTag("t2", "x123")],
+        )
+        assert mv2.version == 2
+
+        def search_versions(filter_string):
+            return [mvd.version for mvd in self.store.search_model_versions(filter_string)]
+
+        assert search_versions(f"name = '{name}' and tag.t2 = 'xyz'") == [1]
```

Comment:
```suggestion
        assert search_versions(f"tag.t2 = 'xyz'") == [1]
```

Do we need `name = '{name}' and `?

Reply:
removed.
but I keep the first one for testing both attribute filter and tag filter existing case.